Signal/src/view controllers/CodeVerificationViewController.m

```diff
@@ -139,6 +139,7 @@ - (void)createViews {
                                                         @"Text field placeholder for SMS verification code during registration");
     _challengeTextField.font = [UIFont ows_lightFontWithSize:21.f];
     _challengeTextField.textAlignment = NSTextAlignmentCenter;
+    _challengeTextField.keyboardType = UIKeyboardTypePhonePad;
```

Comment:
The "registration" view used the "decimal digits" keyboard whereas the "code verification" view used the conventional keyboard.  Both values should only contain decimal digits.  Now both use the "decimal digits" keypad.

I've verified that the layout still works on our largest and smallest form factors.

<img width="346" alt="screen shot 2017-02-16 at 9 37 31 pm" src="https://cloud.githubusercontent.com/assets/625803/23050438/62adbeb4-f490-11e6-97c8-ec76d19653c3.png">
<img width="342" alt="screen shot 2017-02-16 at 9 37 16 pm" src="https://cloud.githubusercontent.com/assets/625803/23050439/62b17b3a-f490-11e6-9b21-b24408988006.png">